packages/interface/src/components/preferences-modal-toggle/feature-toggle.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import PreferencesModalToggle from '../preferences-modal-toggle';
+import { store as interfaceStore } from '../../store';
+
+export default function PreferencesModalFeatureToggle( {
+	scope,
+	feature,
+	...props
+} ) {
+	const isChecked = useSelect(
```

Comment:
The difference between `PreferencesModalFeatureToggle` and `PreferencesModalToggle` is very subtle. Is it worth exposing both components as part of the public API?